frappe/patches/v13_0/update_doctype_default_sort_to_creation.py

```diff
@@ -0,0 +1,8 @@
+import frappe
+
+def execute():
+	custom_doctypes = frappe.get_all('DocType', fields=['name', 'sort_field'])
```

Comment:
You'd want to update `sort_field` only for those that are part of the Frappe App, and are not custom.

Apart from that, even what this patch does is wrong. It makes the site database out of sync with the JSON schema. One `frappe.reload_doctype` will change the `sort_field` value back to what is defined in the DocType's JSON